src/__tests__/patients/notes/NotesTab.test.tsx

```diff
@@ -74,27 +74,4 @@ describe('Notes Tab', () => {
       expect(wrapper.find(components.Modal).prop('show')).toBeTruthy()
     })
   })
-
```

Comment:
there should be tests here that test to make sure that the routing is working properly